port/linux/omrvmem.c

```diff
@@ -1174,13 +1180,35 @@ reserve_memory_with_mmap(struct OMRPortLibrary *portLibrary, void *address, uint
 			useBackingSharedFile = FALSE;
 			Trc_PRT_vmem_reserve_doubleMapAPI_not_available(address, byteAmount);
 		}
+	} else if (useBackingSharedTmpFile) {
+		/* Generate a unique temporary filename from template and open the file. */
+		char filename[FILE_NAME_SIZE + 1];
+		snprintf(
+				filename, sizeof(filename), "omrvmem_%zu_%09d_XXXXXX",
+				(size_t)omrtime_current_time_millis(portLibrary), getpid());
```

Comment:
I see no need for using `omrtime_current_time_millis()` here (particularly with the inappropriate and unnecessary cast).
```c
		snprintffilename, sizeof(filename), "omrvmem_%09d_XXXXXX", getpid());
```